tests/server/master_server/chunk_server_manager_test.cc

```diff
@@ -17,48 +20,637 @@ class ChunkServerManagerTest : public ::testing::Test {
   }
 };
 
-TEST_F(ChunkServerManagerTest, RegisterAndGetChunkServer) {
+// Test helper to create chunkserverlocation
+protos::ChunkServerLocation CreateChunkServerLocation(
```

Comment:
I think I have refactored a function that does this, or a similar one into tests/utils.h. Maybe reuse it if that is the case or move it into this shared util?

Reply:
sure, will use that util some other time.